modules/core/src/main/userId.scala

```diff
@@ -64,6 +65,7 @@ object userId:
   // maybe an Id, maybe a Name... something that's probably cased wrong
   opaque type UserStr = String
   object UserStr extends OpaqueString[UserStr]:
+    given Zero[UserStr] = Zero("")
```

Comment:
I wasn't sure if having Zero not implemented here was intentional

Reply:
It is very much intentional. We never want to have a `UserId("")`, it's not a user ID.